appuio/mariadb-galera/templates/recreate-script.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.forceUpdate }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "%s-sts-deleter" (include "common.names.fullname" .) }}
+  annotations:
+    "helm.sh/hook": pre-upgrade
+    "helm.sh/hook-weight": "-6"
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+data:
+  delete.sh: |
+    #!/bin/bash
+
+    NAME="{{ printf "%s" (include "common.names.fullname" .) }}"
```

Comment:
If you want more useful output from shellcheck, you should consider using lowercase variable names for variables which are not provided to the script through the environment.

With lowercase variables, you get the following for a typo of a variable which is used elsewhere:

```
Line 7:
foundsize=$(echo -En "$fonud" | jq -r '.spec.volumeClaimTemplates[] | select(.metadata.name=="data") | .spec.resources.requests.storage')
                      ^-- SC2154: fonud is referenced but not assigned.
```

while with uppercase variables you get no output for the same typo.

Note that you always get the following warning if you misspell a variable which you only use in a single place

```
Line 5:
FOUND=$(kubectl -n "$namespace" get sts $name -o json)
^-- SC2034: FOUND appears unused. Verify use (or export if used externally).
```